src/index.js

```diff
@@ -114,6 +111,8 @@ exports.init = (Implementation) => {
   configStore.Implementation = Implementation;
   configStore.lianaOptions = opts;
 
+  const schemaFileName = path.join(opts.absoluteRootPath || new ProjectDirectoryUtils().getAbsolutePath(), '.forestadmin-schema.json');
```

Comment:
Exposing a new property `absoluteRootPath` to the agent initialisation is the last thing we want to do.
The less configuration, the easier the integration, the simpler the documentation.

Can you find a robust way to detect that Yarn 2 is used on the project to adapt what `getAbsolutePath` will return in this context?

Reply:
It's cool to have sane defaults but some projects setups are more complex (yarn 2 and yarn workspaces, for the most common?)

It's quite an assumption that it's always possible to have this schema file generated and loaded from a non-configurable path at the root of a project. It can be really cumbersome to play around with this arbitrary constraint.

config and models paths is configurable, that's very cool, why not allow it on this path?

Reply:
Yep I agree with @ArnaudRinquin!

Also in the v7, some environment variables can be passed to the init function, but others are required in process.env because this library uses process.env directly. It's troublesome because I don't necessarily want to use the same variable name than this lib.

Making things configurable would make this lib easier to use.

I also feel like it would be better if we could pass an array of configs instead of a configDir path, but that's another topic.

Reply:
User @merceyz [gave another great reason](https://github.com/ForestAdmin/forest-express/pull/693#discussion_r617468658) for this: with yarn 2 + `enableGlobalCache` the module can be anywhere so... this PR is a simple workaround.